Makefile

```diff
@@ -29,18 +29,20 @@ build: test-data
 	mvn --batch-mode --update-snapshots package
 
 ## test-data
-testDataDir := src/test/resources/
-tempDir := ${testDataDir}temp/
-gitDataDir := ${tempDir}sdk-test-data/
+testDataDir := src/test/resources
```

Comment:
Dropped the trailing `/` from directory name variables as it made it a lot harder (at least for me) to parse what was happening in subsequent commands and also broke my regex strategy to preserve bandit files.